app/javascript/oldjs/automate_import_export.js

```diff
@@ -215,8 +215,28 @@ window.Automate = {
     });
   },
 
-  setUpDefaultGitBranchOrTagValue: function() {
-    $('.git-branch-or-tag').val($('select.git-branches').val());
+  setUpDefaultGitBranchOrTagValue(refType, refName) {
+    if (refType === 'Branch') {
```

Comment:
Like here, why not `refType === 'branch'` ?

Reply:
The UI select component uses the values "Branch" and "Tag" so it won't know what "branch" and "tag" are since the component is case sensitive